app/src/main/res/layout/multiple_choice_interaction_items.xml

```diff
@@ -0,0 +1,30 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="htmlContent"
+      type="String"/>
+  </data>
+  <RelativeLayout
+    android:id="@+id/radio_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="4dp"
+    android:orientation="vertical">
+    <androidx.appcompat.widget.AppCompatRadioButton
+      android:id="@+id/multiple_choice_radio_button"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:padding="4dp"
+      android:focusable="false"
+      app:buttonTint="@color/oppiaDarkBlue"/>
+    <TextView
+      android:id="@+id/tv_multiple_choice_contents"
```

Comment:
id can be : `multiple_choice_content_text_view`

Reply:
Done changes.